components/commercial_filter/filters/gene_ids/gene_ids_filter_view.coffee

```diff
@@ -0,0 +1,54 @@
+_ = require 'underscore'
+Backbone = require 'backbone'
+mediumMap = require './../medium/medium_map.coffee'
+{ numberFormat } = require 'underscore.string'
+
+template = -> require('./index.jade') arguments...
+
+module.exports = class GeneIdsFilterView extends Backbone.View
```

Comment:
Nice start on this filter view! Not sure how complicated this view will get, but I'll take this moment to mention that you should feel free to write new components in ES6 + React, to which I would be happy to pair on setting that up.